server/controllers/constraints/need/create.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
+  pickup_at: {
+    numericality: {
+      greaterThan: Date.now()
+    }
+  },
+  pickup_latitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 90,
+    }
+  },
+  pickup_longitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 180,
+    }
+  },
+  dropoff_latitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 90,
+    }
+  },
+  dropoff_longitude: {
+    presence: true,
+    numericality: {
+      lessThanOrEqualTo: 180,
+    }
+  },
+  requester_name: {
+    length: {
+      minimum: 3,
+    }
+  },
+  requester_phone_number: {
+    length: {
+      minimum: 8,
+    }
+  },
+  cargo_type: {
+    presence: true,
+    type: 'number',
+    inclusion: {
```

Comment:
Wouldn't it be easier to just say it needs to be between 1 and 18?
Same question for next ones

Reply:
It would. Thanks!